packages/salesforcedx-vscode-soql/src/sfdx.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Connection } from '@salesforce/core';
```

Comment:
What is the advantage of having all the services in one file like this ?

Reply:
My idea was to minimize and put together all integration points from the SOQL extensions to the other Salesforce extensions. To reduce code duplication, simplify testing and reduce coupling from the SOQL extension to the other salesforce extensions.

For instance, the logic to load the core extension (which we have to change), is now on a single place instead of four.